install.sh

```diff
@@ -0,0 +1,31 @@
+# Don't run this one directly, use «make install» to run it
```

Comment:
Can you set a shell here? Codacy is also reporting it as an error.

Reply:
Set bash. However have no linux athand now to check if it will still work w/o shell variables which are set in the Makefile.